android_jni/avifandroidjni/src/main/java/com/avif/android/AvifDecoder.java

```diff
@@ -0,0 +1,58 @@
+package com.avif.android;
```

Comment:
I wonder if we should name the package org.aomedia.avif.android, if we consider this as the reference Java implementation of an AVIF decoder from AOMedia.

Reply:
Done.